hugo/layouts/papers/single.html

```diff
@@ -32,6 +32,7 @@
   <meta property="og:site_name" content="ACL Anthology" />
   <meta property="og:url" content="{{ $paper.url }}" />
   <meta property="og:description" content="{{ $paper.author_string }}. {{ $paper.booktitle }}. {{ $paper.year }}." />
+  <link rel="canonical" href="{{ .Params.anthology_id }}" />
```

Comment:
Again, I don't understand this. Shouldn't it be `$paper.url`? The anthology ID is not a URL.

Reply:
sorry, this is my blunder